packages/graphback/src/layers/data/InMemoryDataProvider.ts

```diff
@@ -0,0 +1,35 @@
+import { GraphbackDataProvider } from './GraphbackDataProvider';
+
+
+/**
+ * In Mem data provider used for testing capabilities
+ */
+// TODO
+// tslint:disable-next-line: no-any
+export class InMemoryDataProvider<Type = any, GraphbackContext = any> implements GraphbackDataProvider<Type, GraphbackContext>{
```

Comment:
Ad in-memory database we can use SQLite, but there are also alternatives like https://github.com/techfort/LokiJS that we could take in consideration, or also just write a javascript hash